docs/en-us/dev/user_doc/registry_spi.md

```diff
@@ -0,0 +1,26 @@
+### DolphinScheduler Registry SPI main design
+
+#### How to use it？
+
+First you need to execute the `mvn -U install package -Prelease -Dmaven.test.skip=true` to install the plugin for generating the plugin JAR of the registry. The directory is: dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry.
+
+Next, do the following configuration (using zookeeper as an example)
```

Comment:
"Next, please follow the configuration below (using zookeeper as an example)." should be better.